api/server.go

```diff
@@ -72,7 +72,7 @@ func StartServer(cfg *config.Config) error {
 		return err
 	}
 
-	listeners, err := activation.Listeners(true)
+	listeners, err := activation.Listeners()
```

Comment:
Seems unrelated to this commit

Reply:
After updating dependencies `activation.isteners` changed its signature and does not allow parameters.